cloudscape-design/components
# fix: Disable opening animation for side Split panel

### Description

A month ago we discovered that split panel opening animation is not working due to a typo, the fix was done in this PR https://github.com/cloudscape-design/components/pull/1471 .

After the fix was out we got reports that resizing the split panel is laggy and leaves a right gap while resizing, the issue is reproducible only in React 18 when `createRoot` is used and the page has many components like table.

the issue is due to some excessive rerendering issues, fixing that is not trivial as it needs refactoring different parts of AppLayout and SplitPanel  

After discussing with the team we decided to disable animations back as mitigation for now. 

Related links: AWSUI-22396

### How has this been tested?

- Upgrade to react 18
-  Use createRoot in dev pages
-  visit a dev page with split pane and a table
- before this commit you should see the same bug in the reported ticket

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
